tests/e2e/performance/test_single_pvc_clone_perf.py

```diff
@@ -114,58 +122,70 @@ def test_clone_create_delete_performance(self, interface_type, pvc_size, file_si
+            logger.info(f"Clone number {i+1} creation time is {clone_creation_time_measures[i]} secs.")
+            logger.info(f"Clone number {i+1} creation speed is "
+                        f"{int( int (file_size) / clone_creation_time_measures[i])} MB/sec.")
 
-        logger.info(f"Finished printing {max_num_of_clones} clone creation times "
-                    f"for {pvc_size}GB {interface_type} PVC.")
+        logger.info(f"Finished printing {max_num_of_clones} clone creation times and speed"
+                    f"for {pvc_size} GB {interface_type} PVC with {file_size} MB of data.")
 
-        # deleting one by one and measuring deletion times for each one of the clones create above
+        # deleting one by one and measuring deletion times and speed for each one of the clones create above
         # in case of single clone will run one time
         clone_deletion_time_measures = []
 
-        logger.info(f"Start deleting {max_num_of_clones} clones on {interface_type} PVC of size {pvc_size}GB.")
+        logger.info(f"Start deleting {max_num_of_clones} clones on {interface_type} PVC of size {pvc_size} GB.")
 
         for i in range(max_num_of_clones):
             cloned_pvc_obj = clones_list[i]
             pvc_reclaim_policy = cloned_pvc_obj.reclaim_policy
             cloned_pvc_obj.delete()
-            logger.info(f'Deletion of clone number {i + 1} , the clone name is {cloned_pvc_obj.name}')
+            logger.info(f"Deletion of clone number {i + 1} , the clone name is {cloned_pvc_obj.name}.")
             cloned_pvc_obj.ocp.wait_for_delete(cloned_pvc_obj.name, timeout)
             if pvc_reclaim_policy == constants.RECLAIM_POLICY_DELETE:
                 helpers.validate_pv_delete(cloned_pvc_obj.backed_pv)
             delete_time = helpers.measure_pvc_deletion_time(
                 interface_type, cloned_pvc_obj.backed_pv)
             logger.info(f"Clone number {i + 1} deletion time is {delete_time} secs.")
+
+            deletion_speed = int(int(file_size) / delete_time)
+            logger.info(f"Clone number {i+1} deletion speed is {deletion_speed} MB/sec.")
             clone_deletion_time_measures.append(delete_time)
 
-        logger.info(f"Finished deleting {max_num_of_clones} clones on {interface_type} PVC of size {pvc_size}GB.")
+        logger.info(f"Finished deleting and taking measures for {max_num_of_clones} clones "
+                    f"on {interface_type} PVC of size {pvc_size} GB.")
 
-        logger.info(f"Clone deletion times for {interface_type} PVC of size {pvc_size}GB are:")
+        logger.info(f"Clone deletion time and speed for {interface_type} PVC of size {pvc_size} GB are:")
         for i in range(max_num_of_clones):
             logger.info(f"Clone number {i+1} deletion time is {clone_deletion_time_measures[i]} secs.")
+            logger.info(f"Clone number {i + 1} deletion speed is "
+                        f"{int( int (file_size) / clone_deletion_time_measures[i])} MB/sec.")
```

Comment:
same as above.

Reply:
added an array with dictionary for clone number, deletion time and deletion speed.